src/api/db/student_course_selection.py

```diff
@@ -0,0 +1,37 @@
+class student_course_selection:
+	def __init__(self, db_conn):
+		self.db_conn = db_conn
+
+	def add_selection(self, cid, sem, uid):
+		sql = 	"""
+				INSERT INTO
+					student_course_selection (user_id, semester, course_id)
+				VALUES
+					(%d, %s, %s)
```

Comment:
I believe the psycopg2 page says that you should always use `%s` for the placeholder. It's not like a C format flag where it corresponds to some type.

Reply:
Thank you, will fix for next push.